ql/src/test/results/clientpositive/alterColumnStats.q.out

```diff
@@ -144,35 +144,15 @@ PREHOOK: Input: default@p_n0
 POSTHOOK: query: desc formatted p_n0 c1
 POSTHOOK: type: DESCTABLE
 POSTHOOK: Input: default@p_n0
-col_name            	c1                  	 	 	 	 	 	 	 	 	 	 
-data_type           	string              	 	 	 	 	 	 	 	 	 	 
-min                 	                    	 	 	 	 	 	 	 	 	 	 
```

Comment:
These key-value pairs are removed. Did we ever show any values for these columns, e.g., if stats were available, or were we always printing them empty?

Reply:
Hive either shows the data in the vertical format, which is supposed to be shown in case of having statistics, and the horizontal format with fewer fields, when we don't have the statistics. The previous somewhat messy code produced anomalies like this one, that we didn't have the statistics, still it was shown in vertical format. After the cleanup that small bug is also fixed, like in this case: we don't have statistics, so it is shown in horizontal format.

In other files for example alter_table_column_stats.q.out there are statistics, and thus there is still vertical format present with data.